### ISSUE TITLE
Question: Quotes

### ISSUE THREAD

**Issue by mediaformat** [swicg/activitypub-trust-and-safety#21]
While there are a number of proposals on the mastodon issue queue that deal with this one, as well as some useful discussion on [socialhub](https://socialhub.activitypub.rocks/t/disambiguating-various-interpretations-of-a-quote-feature-pre-fep/3426), I think a discussion here from a trust and safety perspective would be worthwhile.

**Comment by ThisIsMissEm** [comment]
I'd be inclined to say that whilst we can make recommendations regarding trust & safety issues in Quote Posts or Quotes, it is entirely out of scope for this taskforce to define them.

Same goes for reply controls — we can make recommendations to use existing FEPs, but these are outside of what we'd probably be able to specify.

**Comment by mediaformat** [comment]
To be clear, I'm not seeking a definition, or even a recommendation at this stage.

Just a discussion of the T&S merits/drawbacks of current and/or proposed implementations.

**Comment by ThisIsMissEm** [comment]
We'd probably need to revisit this after the Mastodon team have written their Quote Post FEP, since it's covering a lot of trust & safety aspects that existing FEPs don't. All we can really say here is software should implement sufficient controls for actors around whether or not they wish to be quoted, and by whom they can be quoted.

Defining Quote Posts or the pros and cons of different approaches would be out of scope, and for the FEP process and community consensus to align on.

**Comment by ThisIsMissEm** [comment]
Previous notes on Disambiguating various interpretations of a Quote feature — https://socialhub.activitypub.rocks/t/disambiguating-various-interpretations-of-a-quote-feature-pre-fep/3426 (thanks @trwnh for sharing)

**Comment by renchap** [comment]
We (the Mastodon team) are working on a general document about quote posts, which includes a lot of T&S considerations.

It is not ready for sharing right now, but we will publish it once ready.

**Comment by ThisIsMissEm** [comment]
@renchap is there any update on said document?

**Comment by renchap** [comment]
Yes, we published it [3 months ago](https://blog.joinmastodon.org/2025/02/bringing-quote-posts-to-mastodon/): https://github.com/mastodon/specs-background/blob/main/quote-posts/quote-posts-research-and-goals.md

And we also published [FEP-044f](https://codeberg.org/fediverse/fep/src/branch/main/fep/044f/fep-044f.md) for consent-respecting quote posts, and are working on implementing it in Mastodon.

**Comment by ThisIsMissEm** [comment]
Okay, cool, marking this as closed.
